src/manager/evm/manager.rs

```diff
@@ -353,11 +353,43 @@ impl SubnetManager for EthSubnetManager {
 
     async fn set_validator_net_addr(
         &self,
-        _subnet: SubnetID,
-        _from: Address,
-        _validator_net_addr: String,
+        subnet: SubnetID,
+        from: Address,
+        net_addr: String,
+    ) -> Result<()> {
+        let address = contract_address_from_subnet(&subnet)?;
+        log::info!(
+            "set validator net addr: {net_addr:} on evm subnet: {subnet:} at contract: {address:}"
+        );
+
+        let signer = self.get_signer(&from)?;
+        let contract = SubnetActorManagerFacet::new(address, Arc::new(signer));
+
+        let txn = contract.set_validator_net_addr(net_addr);
```

Comment:
Have a look at the latest PR I merged. We need to make the call with the premium estimation to avoid over-paying on the inclusion of the message: https://github.com/consensus-shipyard/ipc-agent/pull/287/files#diff-bea17d53f080353906f6ed16be5c1ed1d4365bad853a9c4db92e8287d59e654fR216

Reply:
Fixed